lib/ddtrace/contrib/rails/core_extensions.rb

```diff
@@ -30,9 +30,9 @@ module CacheStoreFetchExtension
     # It might seem redundant to instrument both read and fetch since
     # fetch very often calls read. But there's no garantee of this, in
     # some cases fetch can call directly read_entry without calling read.
-    def fetch(*args)
+    def fetch(*args, &block)
```

Comment:
This is also the current implementation of a testing app: https://gitlab.com/palazzem/rails3-mysql-demo/blob/master/app/controllers/posts_controller.rb#L21-23